deepinv/models/unet.py

```diff
@@ -92,6 +94,8 @@ def __init__(
         self.compact = scales
         self.Maxpool = nn.MaxPool2d(kernel_size=2, stride=2)
 
+        noiseless = batch_norm != "noiseless"
```

Comment:
This is a bit confusing as it kind of reads `noiseless != "noiseless"`.

Maybe a solution would be to change the `batch_norm` variable for `batch_norm_type` which could be either `None`/`False`, `biasfree` or `True`?

Reply:
That is a good idea, although it would result in breaking changes for those who already have `batch_norm=True` now won't be bias free?

Reply:
Yes but this is not an issue since when loading a model trained in with a different BN than the one used for training it should raise an error.

Reply:
See [440e06b](https://github.com/deepinv/deepinv/pull/137/commits/440e06b0d4c6cd802ade7ad8d6ceecb5fc46b3b1)